src/services/GetServerStatus.ts

```diff
@@ -5,11 +5,11 @@ import {
 } from 'openapi2typescript';
 import { useQuery } from 'react-fetching-library';
 
-import { Operations } from '../generated/OpenapiPrivate';
+import { Operations } from '../generated/OpenapiNotifications';
 import { toServer } from '../types/adapters/ServerAdapter';
 
 const adapter = validationResponseTransformer(
-  (payload: Operations.StatusResourceGetCurrentStatus.Payload) => {
+  (payload: Operations.EventResource$v1GetEvents.Payload) => {
     if (payload.status === 200) {
       return validatedResponse('ServerStatus', 200, toServer(), payload.errors);
     }
```

Comment:
You should also change the validator
```suggestion
    if (payload.status === 200) {
      return validatedResponse(
        'Events',
        200,
        {
          ...payload.value,
          data: payload.value.data.map(toNotificationEvent),
        },
        payload.errors
      );
    }
```